org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -36,17 +36,29 @@ class StateReader {
     }
 }
 
+// Helper method to convert Wink's 0.0-1.0 brightness scale to a 0-100 scale
+function scaleBrightnessToDim(brightnessValue) {
+    return Math.floor(brightnessValue * 100);
+}
+
+// Helper method to convert a 0-100 scale to Wink's 0.0-1.0 brightness scale
+function scaleDimToBrightness(dimmingValue) {
+    return dimmingValue / 100;
+}
+
 // Helper method to convert the device schema to the translator schema.
 function deviceSchemaToTranslatorSchema(deviceSchema) {
     var stateReader = new StateReader(deviceSchema.desired_state, deviceSchema.last_reading);
 
     var powered = stateReader.get('powered');
+    var brightness = stateReader.get('brightness');
 
     return {
         id: deviceSchema['object_type'] + '.' + deviceSchema['object_id'],
```

Comment:
Makes sense. I logged issue 89 to track this.

---
In reply to: [77048026](https://github.com/openT2T/translators/pull/87#discussion_r77048026) [](ancestors = 77048026,76939288)